.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_size = 4
+end_of_line = lf
+
+[*.{c,h,cxx,am,m4}]
+indent_style = tab
```

Comment:
> Or is the goal here not to risk overriding an automatic detection for stuff we don't know about for sure?

Yes, that was the intention. Though as Geany doesn't use the editorconfig file right now it really doesn't matter much and I can change it to whatever else.